css/index.css

```diff
@@ -143,4 +254,98 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+
+nav{
+    float: left;
+    margin: 0;
+    overflow:hidden;
+    text-align: center;
+}
+
```

Comment:
The layout of the navigation links and image lambda-black.png can be done in ~ 8 - 10 lines of CSS.  Essentially, lines 266 - 292 can be removed. and replaced with just nav and nav a.  The way you did it works fine, but the Sprint Readme, specifically says not to use inline-block and floats are not to be used on this sprint.